[ Docs] translated peft (ES) and toctree.yml add

njackman-2344 opened the pull request:
Add the Spanish version of `peft.md` to `transformers/docs/source/es` 

#28936

<!--
Congratulations! You've made it this far! You're not quite done yet though.

Once merged, your PR is going to appear in the release notes with the title you set, so make sure it's a great title that fully reflects the extent of your awesome contribution.

Then, please replace this with a description of the change and which issue is fixed (if applicable). Please also include relevant motivation and context. List any dependencies (if any) that are required for this change.

Once you're done, someone will review your PR shortly (see the section "Who can review?" below to tag some potential reviewers). They may suggest changes to make the code even better. If no one reviewed your PR after a week has passed, don't hesitate to post a new comment @-mentioning the same persons---sometimes notifications get lost.
-->

<!-- Remove if not applicable -->

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu @aaronjimv  If you don't mind reviewing, thanks!! :)

<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @

 If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
 Please tag fewer than 3 people.

Models:

- text models: @ArthurZucker and @younesbelkada
- vision models: @amyeroberts
- speech models: @sanchit-gandhi
- graph models: @clefourrier

Library:

- flax: @sanchit-gandhi
- generate: @gante
- pipelines: @Narsil
- tensorflow: @gante and @Rocketknight1
- tokenizers: @ArthurZucker
- trainer: @muellerzr and @pacman100

Integrations:

- deepspeed: HF Trainer/Accelerate: @pacman100
- ray/raytune: @richardliaw, @amogkam
- Big Model Inference: @SunMarc
- quantization (bitsandbytes, autogpt): @SunMarc and @younesbelkada

Documentation: @stevhliu and @MKhalusova

HF projects:

- accelerate: [different repo](https://github.com/huggingface/accelerate)
- datasets: [different repo](https://github.com/huggingface/datasets)
- diffusers: [different repo](https://github.com/huggingface/diffusers)
- rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)

Maintained examples (not research project or legacy):

- Flax: @sanchit-gandhi
- PyTorch: See Models above and tag the person corresponding to the modality of the example.
- TensorFlow: @Rocketknight1

 -->

njackman-2344 committed:
translated peft es and toctree.yml add

stevhliu commented on docs/source/es/_toctree.yml:
```diff
@@ -96,6 +96,8 @@
     title: Mecanismos de atención
   - local: pad_truncation
     title: Relleno y truncamiento
+    local: peft
```
This should be in the "Tutorial" section.

```suggestion
   - local: peft
```

stevhliu commented on docs/source/es/peft.md:
```diff
@@ -0,0 +1,238 @@
+
+<div class="flex flex-col justify-center">
+  <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/peft/PEFT-hub-screenshot.png"/>
+  <figcaption class="text-center">The adapter weights for a OPTForCausalLM model stored on the Hub are only ~6MB compared to the full size of the model weights, which can be ~700MB.</figcaption>
```
Would be nice to also translate the caption 🙂

stevhliu commented on docs/source/es/peft.md:
```diff
@@ -0,0 +1,238 @@
+## Cargar en 8bit o 4bit
+
+La integración de `bitsandbytes` apoya los tipos de datos precisos que son utilizados para cargar modelos grandes porque
+ guarda memoria (mira la [guia](./quantization#bitsandbytes-integration) de `bitsandbytes` para aprender mas). Agrega el parametro `load_in_8bit` o el parametro `load_in_4bit` al [`~PreTrainedModel.from_pretrained`] y coloca `device_map="auto"` para effectivamente distribuir el modelo tu hardware:
```
The "Quantization" doc doesn't exist in Spanish yet so let's link to the English one for now

```suggestion
 guarda memoria (mira la [guia](https://huggingface.co/docs/transformers/quantization#bitsandbyes) de `bitsandbytes` para aprender mas). Agrega el parametro `load_in_8bit` o el parametro `load_in_4bit` al [`~PreTrainedModel.from_pretrained`] y coloca `device_map="auto"` para effectivamente distribuir el modelo tu hardware:
```

stevhliu reviewed (commented):
Thanks for adding! 🤗

aaronjimv reviewed (commented):
Hi, thanks for adding! The translation in general is good, but I have to point out that it presents several issues with fluency (making it sound natural) and coherence (making it reflect the original content).

njackman-2344 committed:
updated yml and peft translation

aaronjimv reviewed (changes requested):
Hi @njackman-2344, I appreciate you asking me to review again. However, I still have some concerns about the fluency and coherence of the translation. Would you mind taking another look at all the comments made [here](https://github.com/huggingface/transformers/pull/30554/commits/03a1a7d2c0b00ecfde6f4b9c3c19b68ae341b5d7) and addressing those issues? I think it would really make a big difference. Additionally, I wanted to suggest you to use tools like [HuggingChat](https://huggingface.co/chat/) to assist you in the translations and improving their fluency and coherence. Once you've made the necessary changes, I'll be happy to review again.

njackman-2344 commented:
> Hi @njackman-2344, I appreciate you asking me to review again. However, I still have some concerns about the fluency and coherence of the translation. Would you mind taking another look at all the comments made [here](https://github.com/huggingface/transformers/pull/30554/commits/03a1a7d2c0b00ecfde6f4b9c3c19b68ae341b5d7) and addressing those issues? I think it would really make a big difference. Additionally, I wanted to suggest you to use tools like [HuggingChat](https://huggingface.co/chat/) to assist you in the translations and improving their fluency and coherence. Once you've made the necessary changes, I'll be happy to review again.

Ok! Thank you! I didn't see the corrections. I will do that.

njackman-2344 committed:
redid peft.md es translate after notes
